driver-cockroachdb/pom.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This modules should be removed from the latest main. Maybe pull in main changes to this PR?